### ISSUE TITLE
Silent mode

### ISSUE THREAD

**Issue by TheSavior** [facebook/jscodeshift#60]
I'm writing tests for my codemods and thus running jscodeshift repeatedly in my tests. There is a lot of information printed for every run:

```
Processing 1 files...
Spawning 1 workers with 1 files each...
Running in dry mode, no files will be written!
All workers done.
Results: 0 errors 0 unmodifed 0 skipped 1 ok
Time elapsed: 0.902 seconds
```

It would be nice if there was a silent mode. The information printed could be provided as a data object at the end as per #59

**Comment by fkling** [comment]
These are all good ideas :) :+1:

**Comment by cpojer** [comment]
@TheSavior would you mind sending a Pull Request for this? :)

**Comment by TheSavior** [comment]
I can't prioritize working on that currently. @cpojer helped me in my other ticket by showing me the trick to run the transform directly instead of using the runner which got me around this problem for now.

**Comment by fkling** [comment]
Closed with d3bba64464621fb4d3239cac29c4e2a199fe6c35.
